.buildkite/bk.integration.pipeline.yml

```diff
@@ -53,16 +53,41 @@ steps:
           - fleet-privileged
           - upgrade
 
-  - group: "Stateful(Sudo): Ubuntu"
+      - label: "Win2022:non-sudo:{{matrix}}"
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'package-it'
+          .buildkite/scripts/integration-tests.ps1 {{matrix}} false
+        artifact_paths:
+          - build/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-windows-2022"
+        matrix:
+          - default
+
+  - group: "Stateful:Ubuntu"
     key: integration-tests-ubuntu
     depends_on:
-      - package-it
```

Comment:
if we remove this, then the first cmd which downloads from `package-it` might fail.

Reply:
It depends on `integration-ess` that depends on `package-it`

Reply:
I could go either way on this, which I know doesn't help. :) No harm in leaving the dependency there, but it's not a practice we'd necessarily follow if we had a more complicated pipeline with a longer chain of group dependencies.

Reply:
Done